manifest_staging/deploy/secrets-store-csi-driver.yaml

```diff
@@ -28,7 +28,8 @@ spec:
               - /csi-node-driver-registrar
               - --kubelet-registration-path=/var/lib/kubelet/plugins/csi-secrets-store/csi.sock
               - --mode=kubelet-registration-probe
-            initialDelaySeconds: 3
+            initialDelaySeconds: 30
+            timeoutSeconds: 15
```

Comment:
I cannot comment on the resources section but I think that having container limits set to 100m / 100Mi is too little resources for an exec probe to behave deterministically.

@dcantah @kevpar Do we have a rough idea around how much resources are needed to start a new process in a running container? These limits seem too low to accommodate the process set as the container entry point and have something else run periodically.

Reply:
@dcantah @kevpar any thoughts on @marosset's comment?

Reply:
Shouldn't be any more expensive than the same process(es) launched directly on the host. Regardless of that, all of the system services and the init process/any additional execs in the container should be counting towards this 100MB limit set. It might be a bit aggressive depending on what the registrar does. A simple container that just runs `ping -t 127.0.0.1`  consumes about 32-35MB on my machine for example.

Reply:
@marosset Given the `node-driver-registrar` is only in charge for registering the driver should we use 100m/100Mi instead of the current defaults? I know we were using 200m/200Mi for our repro testing but the behavior wasn't deterministic even with that.

Reply:
I'm OK with using 100m/100Mi in this case. (especially with the longer `initialDelaySeconds` for the exec probe) and we can adjust if necessary in the future.

Reply:
sounds good! I've updated the PR to set requests to 100m/100Mi for all the containers. The limits are already 200m/200Mi for the `node-driver-registrar`, `livenessprobe` containers and 400m/400Mi for the `secrets-store` container. PTAL when you get a chance!